include/pingcap/coprocessor/Client.h

```diff
@@ -233,6 +234,7 @@ std::vector<CopTask> buildCopTasks(
     KeyRanges ranges,
     RequestPtr cop_req,
     kv::StoreType store_type,
+    pd::KeyspaceID keyspace_id,
```

Comment:
Add some comments about why we need keyspace_id in `buildCopTasks` but not need for `buildBatchCopTasks`?

Reply:
OK

Reply:
Done